link.txt

```diff
@@ -0,0 +1 @@
+https://code-with-me.global.jetbrains.com/9yrp4ip5iTG2WYxS447IDg#p=IU&fp=57E16C0B74DD6739184C915133589EBBF768559589D92828E29CA48BAE5DB86C&newUi=true
```

Comment:
If you intend to keep this link in the main branch, please add a meaningful name. Additionally, I believe such content should be placed in the CONTRIBUTION.md file.